src/reducer.js

```diff
@@ -28,7 +28,8 @@ function insert (history, state, limit) {
     past: newPast,
     present: state,
     _latestUnfiltered: state,
-    future: []
+    future: [],
+    group: group
```

Comment:
Can you please add the `group` prop to the `StateWithHistory` in `typings.d.ts` interface, also `_latestUnfiltered` would be good to add